ui/components/ui/network-details/index.scss

```diff
@@ -0,0 +1,154 @@
+.network-details {
+  &__wrapper {
+    height: 488px;
+    width: 300px;
+    border-radius: 10px;
+    max-height: 100vh;
+
+    @media screen and (max-width: $break-small) {
+      width: 96%;
+      height: 96%;
+    }
+  }
+
+  &__header {
+    display: flex;
+    width: 175px;
+    height: 32px;
+    background: #f2f3f4;
+    border-radius: 16px;
+    margin-left: 63px;
```

Comment:
`margin-inline-start` might be better for RTL

Reply:
updated

Reply:
Set up KitHub in Thai language as well.